tests/queries/test_explain.py

```diff
@@ -90,6 +90,9 @@ def test_postgres_options(self):
         ]
         if connection.features.is_postgresql_16:
             test_options.append({"generic_plan": True})
+        if connection.features.is_postgresql_17:
+            test_options.append({"memory": True})
+            test_options.append({"serialize": True})
```

Comment:
`memory` is a boolean option and it works fine on PostgreSQL. However, [`SERIALIZE`](https://www.postgresql.org/docs/17/sql-explain.html) should take one of  values `TEXT` or `BINARY`.

Reply:
Thanks for this @felixxm : I applied the feedback in my latest commit.